packages/core/src/domain/configuration/configuration.ts

```diff
@@ -30,43 +30,126 @@ export const TraceContextInjection = {
 export type TraceContextInjection = (typeof TraceContextInjection)[keyof typeof TraceContextInjection]
 
 export interface InitConfiguration {
-  // global options
+  /**
+   * The client token for Datadog. Required for authenticating your application with Datadog.
+   */
   clientToken: string
   beforeSend?: GenericBeforeSendCallback | undefined
+  /**
+   * The percentage of sessions tracked. A value between 0 and 100.
+   * @default 100
+   */
   sessionSampleRate?: number | undefined
+  /**
+   * The percentage of telemetry events sent. A value between 0 and 100.
+   * @default 20
+   */
   telemetrySampleRate?: number | undefined
+  /**
+   * Initialization fails silently if the RUM Browser SDK is already initialized on the page.
+   * @default false
+   */
   silentMultipleInit?: boolean | undefined
+  /**
+   * Allows the use of localStorage when cookies cannot be set. This enables the RUM Browser SDK to run in environments that do not provide cookie support.
+   * See [Monitor Electron Applications Using the Browser SDK](https://docs.datadoghq.com/real_user_monitoring/guide/monitor-electron-applications-using-browser-sdk) for further information.
+   * @default false
+   */
   allowFallbackToLocalStorage?: boolean | undefined
+  /**
+   * Allow untrusted events to be collected by the browser SDK. Can be useful if you heavily rely on programmatic events, such as in an automated UI test environment.
```

Comment:
**💬 suggestion:** ‏
```suggestion
   * Allow listening to DOM events dispatched programmatically (untrusted events). Enabling this option can be useful if you heavily rely on programmatic events, such as in an automated UI test environment.
```

I would also add a link to https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted